src/fastoad/cmd/fast.py

```diff
@@ -275,6 +274,12 @@ def run(self):
             action="store_true",
             help="shows detailed information for each system",
         )
+        parser_list_modules.add_argument(
+            "source_path",
+            nargs="?",
```

Comment:
For the command to accept several arguments, you have to use:
```suggestion
            nargs="*",
```
See https://docs.python.org/3/library/argparse.html#nargs

Reply:
Corrected in ff70f99